### DISCUSSION TITLE
Cannot install Extensions

### DISCUSSION THREAD

**Discussion by ronkristoff** [zed-industries/zed#9618] [Feature Requests]
I am trying to install an extension, but when I click Install nothing is happening, when the button disables the goes back.
I'm using Macbook Pro Sonoma 14.2.1n

**Comment by Moshyfawn** [comment]
What is the extension you are trying to install (or is it reproducible with any extension?) and what version of Zed are you using?

**Reply by ronkristoff** [reply]
It's actually any kind of extension. I'm using Zed 0.127.3

here is a screen record of what's happening.

https://github.com/zed-industries/zed/assets/49369032/91e19d0c-82b9-4f1f-b72b-f92677fa327b

**Reply by Moshyfawn** [reply]
Would you be able to click install on any of them, then open the command palette and select `zed: open log`; see if there are any extension-related errors at the bottom of the log file.

**Reply by ronkristoff** [reply]
i am getting this message:
2024-03-21T12:10:23+08:00 [INFO] installing extension new-darcula 1.0.3
2024-03-21T12:10:23+08:00 [WARN] request completed with error: failed to resolve host name
2024-03-21T12:10:23+08:00 [ERROR] crates/extension/src/extension_store.rs:461: error downloading extension: failed to resolve host name

**Comment by JosephTLyons** [comment]
Hey @ronkristoff, can you try the latest build of whatever release channel you are on and see if the issue is fixed for you?

[Stable](https://zed.dev/releases/stable)
[Preview](https://zed.dev/releases/preview)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
